### DISCUSSION TITLE
Adding new page in Hubs

### DISCUSSION THREAD

**Discussion by jasoncks** [mozilla/hubs#3471] [Hubs Cloud Questions] [answered]
I am try to add some html pages in hubs.

I modify the webpack.config.js and the pages can be accessed after I run "npm run dev" using the links as below:
https://localhost:8080/aaa.html
https://localhost:8080/bbb.html

But when I run "npm run deploy" to deploy the files to AWS.
When I enter the links
https://www.{my-hubs-domain}.com/aaa.html
https://www.{my-hubs-domain}.com/bbb.html

Invalid URL will be shown.

Is there any step I missed? 
What other files should I need to modify?

**Comment by robinkwilson** [comment, marked as answer]
This is an open issue in the hubs-cloud repo! https://github.com/mozilla/hubs-cloud/issues/106

Aka: it hasn't been implemented yet.

**Reply by robinkwilson** [reply]
Thumbs up the issue to vote to increase priority!
